src/libs/API/parameters/ResolveDuplicatesParams.ts

```diff
@@ -1,6 +1,6 @@
 type ResolveDuplicatesParams = {
     /** The ID of the transaction that we want to keep */
-    transactionID: string;
+    transactionID: string | undefined;
```

Comment:
Are you sure that the api call can be called without valid `transactionID`?

Reply:
Checked it and looks fine when I am passing undefined I am getting same error from API when passing empty string, which say that there are missing information to proceed